### ISSUE TITLE
Unable to use the module in Terraform

### ISSUE THREAD

**Issue by nikoul** [terraform-aws-modules/terraform-aws-rds-aurora#9]
Hi,

I'm not able to use your module in terraform. This is the first time this happens to me with a terraform module.

Here is the result of a terraform get command :
```
Initializing modules...

module.security_group-db
Found version 2.7.0 of terraform-aws-modules/security-group/aws on registry.terraform.io
Getting source "terraform-aws-modules/security-group/aws"
module.aurora
Found version 0.0.1 of terraform-aws-modules/rds-aurora/aws on registry.terraform.io
Getting source "terraform-aws-modules/rds-aurora/aws"
Error downloading modules: Error loading modules: module aurora: No Terraform configuration files found in directory: .terraform/modules/3a6b6096207200690f118584dd370e00/terraform-aws-modules-terraform-aws-rds-aurora-7db32bb
```
When I go in the temporary folder, there is only a README file :

```
# terraform-aws-rds-aurora
Terraform module which creates RDS Aurora resources on AWS

WORK IN PROGRESS
```

Can you help me ?
Thanks

**Comment by max-rocket-internet** [comment]
I don't really know how the registry works. You can ask @antonbabenko.

But you can also refer to this module like this:

```
module "db" {
  source    = "git@github.com:terraform-aws-modules/terraform-aws-rds-aurora.git?ref=v0.0.1"
...
```

**Comment by robertomoutinho** [comment]
I've tried the registry format and the above github format also... none of them seems to be working.

The only working source url was =>

`source = "github.com/terraform-aws-modules/terraform-aws-rds-aurora.git"`

**Comment by antonbabenko** [comment]
Thanks @nikoul for opening this issue!

The problem was that there were no real releases in the registry, I have just released [v1.0.0](https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/releases/v1.0.0) and it appears correctly - https://registry.terraform.io/modules/terraform-aws-modules/rds-aurora/aws/1.0.0

You can now use it like this: `source = "terraform-aws-modules/rds-aurora/aws"`
